src/App.tsx

```diff
@@ -1,14 +1,69 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const [listOfTodos, setTodoList] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState('all');
+  const [serchQuery, setSearchQuery] = useState('');
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [isLoading, setIsLoading] = useState(true);
+
+  enum Filter {
+    All = 'all',
+    Completed = 'completed',
+    Incomplete = 'incomplete',
+  }
+
+  const filterTodos = () => {
+    let filteredTodos: Todo[] = [...listOfTodos];
+
+    if (filter !== Filter.All) {
+      filteredTodos = filteredTodos.filter(todo => {
+        return (
+          filter === Filter.Completed
+            ? todo.completed
+            : !todo.completed
+        );
+      });
+    }
```

Comment:
In my opinion switch(filter) will be more flexible

Reply:
using switch will require more code to achieve the same result as the current code

Reply:
But it will be more flexible in case you will have more Filter options